src/category_theory/types.lean

```diff
@@ -170,3 +170,17 @@ def to_equiv (i : X ≅ Y) : X ≃ Y :=
 @[simp] lemma to_equiv_symm_fun (i : X ≅ Y) : (i.to_equiv.symm : Y → X) = i.inv := rfl
 
 end category_theory.iso
+
+universe u
+
+-- We prove `equiv_iso_iso` and then use that to sneakily construct `equiv_equiv_iso`.
+-- (In this order the proofs are handled by `obviously`.)
+
+/-- equivalences (between types in the same universe) are the same as (isomorphic to) isomorphisms of types -/
+def equiv_iso_iso {X Y : Sort u} : (X ≃ Y) ≅ (X ≅ Y) :=
```

Comment:
I would expect some simp-lemmas for the projections of the definitions... does it make sense to add those?

Reply:
Added. Also, made the arguments to `equiv_equiv_iso` explicit, because otherwise the coercion to function won't fire (and added a comment to this effect).

Reply:
They weren't added on my version of Github... did you push?

Reply:
Sorry, hopefully all there now.